fpdf/fpdf.py

```diff
@@ -400,6 +400,13 @@ def write_html(self, text, *args, **kwargs):
             heading_sizes (dict): font size per heading level names ("h1", "h2"...)
             pre_code_font (str): font to use for <pre> & <code> blocks
             warn_on_tags_not_matching (bool): control warnings production for unmatched HTML tags
+            element_colors (dict): dictionary of colors for elements. Possible keys:
+            * "link" sets the link color for a href tag
+            * "li" sets the bullet or number for <li> tags
+            * "blockquote" sets the color for <blockquote> tags
+            * "headings" sets the color is not specified in a h1, h2 etc. heading tag
```

Comment:
Something's not right with the spelling/grammar here.
Do you mean "**_if_** not specified" ?